bgc_derivation/processo2.m

```diff
@@ -135,8 +138,36 @@ function processo2(floatnos,varargin)
           doxycalc  =  molar_doxy;             % Get right dimensions
           doxycalc(mask)  =  O2ctoO2s(molar_doxy(mask),T(mask),S(mask),P(mask)); %,p_atm(mask));
           doxy  =  ncdox{'DOXY'}(:)';
+        
+        case '201_202_202'
+          T  =  ncctd{'TEMP'}(:);
+          S  =  ncctd{'PSAL'}(:);
+          P  =  ncctd{'PRES'}(:);
+          bphasedoxy=ncdox{'BPHASE_DOXY'}(:);
+          rphasedoxy=ncdox{'RPHASE_DOXY'}(:);
+          doxy  =  ncdox{'DOXY'}(:);
+          
+          %retrieving the calibration coeffs.
+          stc  =  stcoeff(1).optode2; % could be many optodes.sgl working on geto2sensor to solve the issue.         
+          maskctdS = S == 99999; % as defined in the .nc file
```

Comment:
maskctdS and mask not used anymore, perhaps comment or delete.

Reply:
I used the mask for just the salinity. If you think that is not the right approach, I will remove that.

Reply:
I mean that the maskctdS and mask variable doesn't seem to be used further down the code, so if redundant then probably the mastkctdS and mask variable can be deleted.